airbyte-integrations/connectors/source-google-search-console/unit_tests/unit_test.py

```diff
@@ -198,13 +198,15 @@ def test_check_connection(config_gen, config, mocker, requests_mock):
     )
 
     # test custom_reports
-    assert command_check(source, config_gen(custom_reports="")) == AirbyteConnectionStatus(
-        status=Status.FAILED,
-        message="\"Unable to check connectivity to Google Search Console API - Exception('custom_reports is not valid JSON')\"",
-    )
-    assert command_check(source, config_gen(custom_reports="{}")) == AirbyteConnectionStatus(
-        status=Status.FAILED, message="'<ValidationError: \"{} is not of type \\'array\\'\">'"
-    )
+    with pytest.raises(AirbyteTracedException):
```

Comment:
can we add unit tests for `config_migration`?

Reply:
definitely, the unit tests will be addeed for  sure.